plugins/inputs/unbound/README.md

```diff
@@ -0,0 +1,139 @@
+# Unbound Input Plugin
+
+This plugin gathers stats from [Unbound - a validating, recursive, and caching DNS resolver](https://www.unbound.net/)
+
+### Configuration:
+
+```toml
+ # A plugin to collect stats from Unbound - a validating, recursive, and caching DNS resolver
+ [[inputs.unbound]]
+   ## If running as a restricted user you can prepend sudo for additional access:
+   #use_sudo = false
+
+   ## The default location of the unbound-control binary can be overridden with:
+   binary = "/usr/sbin/unbound-control"
+
+   # The default timeout of 1s can be overriden with:
+   #timeout = "1s"
+
+  ## By default, telegraf gather stats for 4 metric points.
+  ## Setting stats will override the defaults shown below.
+  ## Glob matching can be used, ie, stats = ["total.*"]
+  ## stats may also be set to ["*"], which will collect all stats
+  ## except histogram.* statistics that will never be collected.
+  stats = ["total.*", "num.*","time.up", "mem.*"]
```

Comment:
I think we should remove this filter, users can simply use the builtin fielddrop/fieldpass filters to get the same behavior.